tasks/setup_environment.yml

```diff
@@ -2,7 +2,7 @@
 - name: Set project and service identifier in facts
   ansible.builtin.set_fact:
     project: "{{ group_names[0] }}"
-    service_identifier: "{{ group_names[0] }}{{ '' if type in default_node_types else '-' + type }}"
+    service_identifier: "{{ group_names[0] }}{{ '' if type in default_node_types else '-' + type }}{{ '-engine' if engine is defined and engine else '' }}"
```

Comment:
I think from the business point of view, we wont have to run a fullnode for this, so we can assign types as node, and engine, instead of validator and fullnode